editor/import/resource_importer_obj.cpp

```diff
@@ -188,7 +188,7 @@ static Error _parse_material_library(const String &p_path, Map<String, Ref<Spati
 	return OK;
```

Comment:
very tolerant and nice commnuity imo. run by great leadership skills, you should be proud juan and akien for leading this great floss project!
![](https://i.gyazo.com/2fd0c035013b3c1bb0652552d35fde92.png)
![](https://i.gyazo.com/e0acd22258872330f342d5d189564f1e.png)
![](https://i.gyazo.com/81b238ce283aa55126e3a2f608583837.png)